### ISSUE TITLE
[BUG] No output outside a terminal

### ISSUE THREAD

**Issue by Locorock** [krathalan/wtwitch#13]
**Describe the bug**
No output is produced when any command is executed outside a terminal

**To Reproduce**
wtwitch c | rofi -dmenu, executed with an i3 keybind

**Expected behavior**
I expected rofi to show me all the rows of wtwitch c as options, but i got nothing

**Logs/system info (please complete the following information):**
 - OS: Manjaro
 - `wtwitch` version: 2.5.1

**Additional context**
Having the command be executed within a terminal solves the issue (but doing so opens a new window)

**Comment by Locorock** [comment]
ok yeah apparently the script checks terminal width to cut titles and such, so when a terminal isn't actually nothing works because there is no terminal width. Managed to fix it by just removing title-cutting from the code but there should be a way to have it only affect instances not run in a terminal.

**Comment by Locorock** [comment]
that's one way of solving it i guess, really convoluted tho

**Comment by falsified** [comment]
yeah, not too bad, tweaked it to get it working right, so it'll exit out of the terminal called

dmenu -> script1
script:1 call term executing script2
script2:  wtwitch c >tmpfile, grep, sed, into dmenu dialog -> select streamer and hit enter -> launches streamlnk with mpv via nohup, so parent terminal doesnt close out new spawn, bonus, set tray icon sleep and script1 exits spawned term

Lemme know if you'd want the code, it would only  take a tiny bit of modifications

**Comment by Locorock** [comment]
> yeah, not too bad, tweaked it to get it working right, so it'll exit out of the terminal called
> 
> dmenu -> script1 script:1 call term executing script2 script2: wtwitch c >tmpfile, grep, sed, into dmenu dialog -> select streamer and hit enter -> launches streamlnk with mpv via nohup, so parent terminal doesnt close out new spawn, bonus, set tray icon sleep and script1 exits spawned term
> 
> Lemme know if you'd want the code, it would only take a tiny bit of modifications

Yeah i already edited the code of wtwitch by simply removing the line that cuts text short. Should probably add a config for that and fork this repo
